client/flutter/lib/pages/onboarding/onboarding_page.dart

```diff
@@ -1,9 +1,16 @@
+import 'package:WHOFlutter/api/user_preferences.dart';
+import 'package:WHOFlutter/pages/home_page.dart';
 import 'package:WHOFlutter/pages/onboarding/legal_landing_page.dart';
 import 'package:WHOFlutter/pages/onboarding/location_sharing_page.dart';
 import 'package:WHOFlutter/pages/onboarding/notifications_page.dart';
+import 'package:firebase_analytics/firebase_analytics.dart';
 import 'package:flutter/material.dart';
 
 class OnboardingPage extends StatefulWidget {
+  const OnboardingPage({Key key, @required this.analytics}) : super(key: key);
```

Comment:
Can you remove `analytics` from `OnboardingPage`? I don't believe we're using it.

Reply:
https://github.com/WorldHealthOrganization/app/blob/7bc51a5e857b8cada589cf5f4904a585aaa80252/client/flutter/lib/pages/onboarding/onboarding_page.dart#L71
It's being supplied to the home page